src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -4,34 +4,45 @@ import { EventDelegator } from '../Rendering/EventDelegator';
 
 let hasEnabledNavigationInterception = false;
 let hasRegisteredNavigationEventListeners = false;
+let hasLocationChangingListeners = false;
```

Comment:
Nit: The formatting in this file needs to be corrected.

Reply:
Can you elaborate? I am not sure what formatting you want changed.

Reply:
I noticed a few minor things like the braces on L91 to L100, L145, and some additional whitespace on L134.